packages/adapter-xata/src/index.ts

```diff
@@ -0,0 +1,113 @@
+import { Adapter } from "next-auth/src/adapters"
```

Comment:
```suggestion
import type { Adapter } from "next-auth/adapters"
```

Reply:
Right! You need to run `pnpm build` inside `packages/next-auth`, so it get's generated.

Reply:
Awesome! Thanks! Done!